index.md

```diff
@@ -17,71 +17,68 @@ FeatureBase Help contains high level overviews of software functionality, plus p
 
 FeatureBase is a feature-oriented database platform that powers real-time analytics and machine learning applications by executing low-latency, high-throughput, and highly concurrent workloads. Similar to the evolution of data formats from row-oriented to columnar, FeatureBase further evolves columnar into a feature-oriented format that makes each distinct data value individually addressable (accessible, readable, writable and retrievable).
 
+FeatureBase is similar to a columnar store, but breaks each column into each of its unique values so that they can be represented as a single bit. This data representation is excellent for a variety of analytical workloads.
+
 Our novel approach minimizes I/O on queries by allowing the database engine to read and write exactly the data it needs and intelligently compress that data in memory. The result is a step-function improvement in analytical workloads.
 
 * [Learn more on featurebase.com](https://featurebase.com)
 
-## Before you begin
-
-* [Learn more about FeatureBase](https://www.featurebase.com/){:target="_blank"}
-* [Contact FeatureBase support](https://www.featurebase.com/contact-us){:target="_blank"}
-
 ## Comparing Cloud and Community editions
 
 These tables provided a side-by-side comparison of Cloud and Community functionality.
 
 ## Infrastructure
 
-| Feature | Cloud  | Community  | Notes  |
-| ------ | ----- | ----------- | ----------- |
-| Click to Deploy |  Yes | No | Windows installations require a Hypervisor  |
+| Feature | Cloud | Community |
+|---|---|---|
+| Deployment | [Cloud deployment](/docs/cloud/cloud-signup) | [Community installation](/docs/community/com-home) |
+| Regional Deployment | No | Yes |  |
+<!-- Removed because neither product supports, therefore it's not useful
```

Comment:
It is useful because people always ask these questions and our answer (for now) is no. Please return